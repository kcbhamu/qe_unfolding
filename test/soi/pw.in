&CONTROL
  calculation = 'scf',
  outdir = './',
  prefix = 'graphene',
  pseudo_dir = './',
  restart_mode = 'from_scratch',
/
&SYSTEM
  ecutwfc = 45.0,
  ibrav = 0,
  nat = 8,
  ntyp = 1,
  lspinorb = .TRUE.,
  noncolin = .TRUE.,
  occupations = 'smearing',
  smearing = 'mp',
  degauss = 0.02
  starting_magnetization = 0.0,
/
&ELECTRONS
  conv_thr = 1d-08,
/
ATOMIC_SPECIES
  C  12.0107   C.upf
ATOMIC_POSITIONS {crystal}
  C 0.166667 0.166667 0.000000
  C 0.166667 0.666667 0.000000
  C 0.666667 0.166667 0.000000
  C 0.666667 0.666667 0.000000
  C 0.333333 0.333333 0.000000
  C 0.333333 0.833333 0.000000
  C 0.833333 0.333333 0.000000
  C 0.833333 0.833333 0.000000
CELL_PARAMETERS {angstrom}
  4.920000 0.000000 0.000000
  2.460000 4.260845 0.000000
  0.000000 0.000000 20.000000
K_POINTS automatic
12 12 1 0 0 0
