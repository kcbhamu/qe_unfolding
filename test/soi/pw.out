
     Program PWSCF v.6.4.1 starts on 15Dec2019 at 15: 4:59 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on     320 processor cores
     Number of MPI processes:               320
     Threads/MPI process:                     1

     MPI processes distributed on    10 nodes
     K-points division:     npool     =      10
     R & G space division:  proc/nbgrp/npool/nimage =      32
     Reading input from pw.in

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     one sub-group per band group will be used
     scalapack distributed-memory algorithm (size of sub-group:  4*  4 procs)

     Found symmetry operation: I + (  0.0000 -0.5000  0.0000)
     This is a supercell, fractional translations are disabled
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          33      33      9                 3595     3595     553
     Max          34      34     10                 3614     3614     574
     Sum        1069    1069    313               115397   115397   18123
 


     bravais-lattice index     =            0
     lattice parameter (alat)  =       9.2975  a.u.
     unit-cell volume          =    2829.3550 (a.u.)^3
     number of atoms/cell      =            8
     number of atomic types    =            1
     number of electrons       =        32.00
     number of Kohn-Sham states=           40
     kinetic-energy cutoff     =      45.0000  Ry
     charge density cutoff     =     180.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)
     Non magnetic calculation with spin-orbit


     celldm(1)=   9.297453  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.500000   0.866025   0.000000 )  
               a(3) = (   0.000000   0.000000   4.065041 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000 -0.577350  0.000000 )  
               b(2) = (  0.000000  1.154701  0.000000 )  
               b(3) = (  0.000000  0.000000  0.246000 )  


     PseudoPot. # 1 for C  read from file:
     ./C.upf
     MD5 check sum: d80ec9912146af6c4c1760317d1f95e9
     Pseudo is Norm-conserving + core correction, Zval =  4.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1248 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   1
                l(6) =   1

     atomic species   valence    mass     pseudopotential
        C              4.00    12.01070     C ( 1.00)

     24 Sym. Ops., with inversion, found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           C   tau(   1) = (   0.2500005   0.1443379   0.0000000  )
         2           C   tau(   2) = (   0.5000005   0.5773506   0.0000000  )
         3           C   tau(   3) = (   0.7500005   0.1443379   0.0000000  )
         4           C   tau(   4) = (   1.0000005   0.5773506   0.0000000  )
         5           C   tau(   5) = (   0.4999995   0.2886748   0.0000000  )
         6           C   tau(   6) = (   0.7499995   0.7216876   0.0000000  )
         7           C   tau(   7) = (   0.9999995   0.2886748   0.0000000  )
         8           C   tau(   8) = (   1.2499995   0.7216876   0.0000000  )

     number of k points=    19  Methfessel-Paxton smearing, width (Ry)=  0.0200
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0069444
        k(    2) = (   0.0000000   0.0962250   0.0000000), wk =   0.0416667
        k(    3) = (   0.0000000   0.1924501   0.0000000), wk =   0.0416667
        k(    4) = (   0.0000000   0.2886751   0.0000000), wk =   0.0416667
        k(    5) = (   0.0000000   0.3849002   0.0000000), wk =   0.0416667
        k(    6) = (   0.0000000   0.4811252   0.0000000), wk =   0.0416667
        k(    7) = (   0.0000000  -0.5773503   0.0000000), wk =   0.0208333
        k(    8) = (   0.0833333   0.1443376   0.0000000), wk =   0.0416667
        k(    9) = (   0.0833333   0.2405626   0.0000000), wk =   0.0833333
        k(   10) = (   0.0833333   0.3367877   0.0000000), wk =   0.0833333
        k(   11) = (   0.0833333   0.4330127   0.0000000), wk =   0.0833333
        k(   12) = (   0.0833333  -0.6254628   0.0000000), wk =   0.0833333
        k(   13) = (   0.1666667   0.2886751   0.0000000), wk =   0.0416667
        k(   14) = (   0.1666667   0.3849002   0.0000000), wk =   0.0833333
        k(   15) = (   0.1666667  -0.6735753   0.0000000), wk =   0.0833333
        k(   16) = (   0.1666667  -0.5773503   0.0000000), wk =   0.0416667
        k(   17) = (   0.2500000  -0.7216878   0.0000000), wk =   0.0416667
        k(   18) = (   0.2500000  -0.6254628   0.0000000), wk =   0.0833333
        k(   19) = (   0.3333333  -0.5773503   0.0000000), wk =   0.0138889

     Dense  grid:   115397 G-vectors     FFT dimensions: (  40,  40, 162)

     Estimated max dynamical RAM per process >      14.52 MB

     Estimated total dynamical RAM >       4.54 GB

     Initial potential from superposition of free atoms

     starting charge   31.99964, renormalised to   32.00000

     negative rho (up, down):  1.096E-03 0.000E+00
     Starting wfcs are   64 randomized atomic wfcs

     total cpu time spent up to now is        5.1 secs

     Self-consistent Calculation

     iteration #  1     ecut=    45.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.2

     negative rho (up, down):  2.161E-04 0.000E+00

     total cpu time spent up to now is        8.0 secs

     total energy              =     -96.05624306 Ry
     Harris-Foulkes estimate   =     -96.50468146 Ry
     estimated scf accuracy    <       0.76187243 Ry

     iteration #  2     ecut=    45.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.38E-03,  avg # of iterations =  3.2

     negative rho (up, down):  4.049E-05 0.000E+00

     total cpu time spent up to now is        8.8 secs

     total energy              =     -96.18236668 Ry
     Harris-Foulkes estimate   =     -96.18291876 Ry
     estimated scf accuracy    <       0.00981002 Ry

     iteration #  3     ecut=    45.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.07E-05,  avg # of iterations = 10.6

     total cpu time spent up to now is        9.6 secs

     total energy              =     -96.18453380 Ry
     Harris-Foulkes estimate   =     -96.18446662 Ry
     estimated scf accuracy    <       0.00070500 Ry

     iteration #  4     ecut=    45.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.20E-06,  avg # of iterations =  5.6

     total cpu time spent up to now is       10.2 secs

     total energy              =     -96.18459752 Ry
     Harris-Foulkes estimate   =     -96.18459599 Ry
     estimated scf accuracy    <       0.00000168 Ry

     iteration #  5     ecut=    45.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  5.25E-09,  avg # of iterations =  4.0

     total cpu time spent up to now is       10.7 secs

     total energy              =     -96.18460299 Ry
     Harris-Foulkes estimate   =     -96.18460302 Ry
     estimated scf accuracy    <       0.00000021 Ry

     iteration #  6     ecut=    45.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  6.56E-10,  avg # of iterations =  2.0

     total cpu time spent up to now is       11.1 secs

     total energy              =     -96.18460304 Ry
     Harris-Foulkes estimate   =     -96.18460306 Ry
     estimated scf accuracy    <       0.00000004 Ry

     iteration #  7     ecut=    45.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.32E-10,  avg # of iterations =  2.0

     total cpu time spent up to now is       11.4 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 ( 14337 PWs)   bands (ev):

   -21.9851 -21.9851 -16.7064 -16.7064 -16.7064 -16.7064 -16.7063 -16.7063
   -15.6713 -15.6713 -15.6712 -15.6712 -15.6712 -15.6712 -10.0008 -10.0008
    -8.7753  -8.7753  -8.7753  -8.7753  -8.7752  -8.7752  -5.3227  -5.3227
    -5.3140  -5.3140  -4.6841  -4.6841  -4.6841  -4.6841  -4.6840  -4.6840
    -0.6029  -0.6029  -0.6028  -0.6028  -0.6028  -0.6028   0.9115   0.9115

          k = 0.0000 0.0962 0.0000 ( 14343 PWs)   bands (ev):

   -21.9450 -21.9450 -17.2168 -17.2168 -16.8473 -16.8473 -16.8472 -16.8472
   -15.4605 -15.4605 -15.4604 -15.4604 -15.0989 -15.0989  -9.9552  -9.9552
    -8.9026  -8.9026  -8.9025  -8.9025  -8.7330  -8.7330  -5.4964  -5.4964
    -5.4140  -5.4140  -4.9392  -4.9392  -4.7134  -4.7134  -4.7133  -4.7133
    -0.5260  -0.5260  -0.5259  -0.5259  -0.2817  -0.2817   0.9675   0.9675

          k = 0.0000 0.1925 0.0000 ( 14383 PWs)   bands (ev):

   -21.8211 -21.8211 -17.9784 -17.9784 -17.1073 -17.1073 -17.1072 -17.1072
   -14.9995 -14.9995 -14.9994 -14.9994 -14.1514 -14.1514  -9.8065  -9.8065
    -9.2717  -9.2717  -9.2716  -9.2716  -8.6065  -8.6065  -5.9967  -5.9967
    -5.6701  -5.6701  -5.5396  -5.5396  -4.7934  -4.7934  -4.7934  -4.7934
    -0.3207  -0.3207  -0.3206  -0.3206   0.5113   0.5113   1.1355   1.1355

          k = 0.0000 0.2887 0.0000 ( 14415 PWs)   bands (ev):

   -21.6154 -21.6154 -18.7125 -18.7125 -17.3464 -17.3464 -17.3464 -17.3464
   -14.4322 -14.4322 -14.4321 -14.4321 -13.1147 -13.1147  -9.8066  -9.8066
    -9.8065  -9.8065  -9.5606  -9.5606  -8.4076  -8.4076  -6.7498  -6.7498
    -6.2561  -6.2561  -6.0472  -6.0472  -4.8926  -4.8926  -4.8925  -4.8925
    -0.0359  -0.0359  -0.0358  -0.0358   1.4156   1.4156   1.5385   1.5385

          k = 0.0000 0.3849 0.0000 ( 14449 PWs)   bands (ev):

   -21.3273 -21.3273 -19.3842 -19.3842 -17.5274 -17.5274 -17.5274 -17.5274
   -13.8195 -13.8195 -13.8195 -13.8195 -12.0302 -12.0302 -10.4374 -10.4374
   -10.4374 -10.4374  -9.2179  -9.2179  -8.1307  -8.1307  -7.6726  -7.6726
    -6.9764  -6.9764  -6.4844  -6.4844  -4.9834  -4.9834  -4.9833  -4.9833
     0.2548   0.2548   0.2549   0.2549   1.8076   1.8076   2.5946   2.5947

          k = 0.0000 0.4811 0.0000 ( 14426 PWs)   bands (ev):

   -20.9581 -20.9581 -19.9830 -19.9830 -17.6380 -17.6380 -17.6380 -17.6380
   -13.2367 -13.2367 -13.2366 -13.2366 -11.0445 -11.0445 -11.0444 -11.0444
   -10.9190 -10.9190  -8.7819  -8.7819  -8.7061  -8.7061  -7.7828  -7.7828
    -7.6510  -7.6510  -6.9396  -6.9396  -5.0450  -5.0450  -5.0450  -5.0450
     0.4761   0.4761   0.4761   0.4761   2.3112   2.3112   3.1023   3.1023

          k = 0.0000-0.5774 0.0000 ( 14460 PWs)   bands (ev):

   -20.5094 -20.5094 -20.5093 -20.5093 -17.6767 -17.6767 -17.6767 -17.6767
   -12.9353 -12.9353 -12.9353 -12.9353 -11.3594 -11.3594 -11.3594 -11.3594
    -9.8014  -9.8014  -9.8013  -9.8013  -8.2564  -8.2564  -8.2564  -8.2564
    -7.3831  -7.3831  -7.3831  -7.3831  -5.0671  -5.0671  -5.0671  -5.0671
     0.5577   0.5577   0.5577   0.5577   2.9257   2.9257   2.9257   2.9257

          k = 0.0833 0.1443 0.0000 ( 14396 PWs)   bands (ev):

   -21.8624 -21.8624 -17.5693 -17.5693 -17.5693 -17.5693 -16.5994 -16.5994
   -15.5655 -15.5655 -14.6179 -14.6179 -14.6178 -14.6178  -9.8568  -9.8568
    -9.3148  -9.3148  -8.8224  -8.8224  -8.8223  -8.8223  -5.8243  -5.8243
    -5.5953  -5.5953  -5.1786  -5.1786  -5.1786  -5.1786  -4.5329  -4.5329
    -0.6402  -0.6402   0.0615   0.0615   0.0617   0.0617   1.0795   1.0795

          k = 0.0833 0.2406 0.0000 ( 14417 PWs)   bands (ev):

   -21.6976 -21.6976 -18.3239 -18.3239 -17.8676 -17.8676 -16.6327 -16.6327
   -15.2596 -15.2596 -14.0618 -14.0618 -13.6199 -13.6199  -9.8953  -9.8953
    -9.6595  -9.6595  -9.1454  -9.1454  -8.6565  -8.6565  -6.4291  -6.4291
    -5.9340  -5.9340  -5.8582  -5.8582  -5.3999  -5.3999  -4.4159  -4.4159
    -0.6216  -0.6216   0.4633   0.4633   0.9938   0.9938   1.3036   1.3036

          k = 0.0833 0.3368 0.0000 ( 14413 PWs)   bands (ev):

   -21.4506 -21.4506 -19.0277 -19.0277 -18.0949 -18.0949 -16.7785 -16.7785
   -14.7203 -14.7203 -13.4594 -13.4594 -12.5603 -12.5603 -10.5779 -10.5779
    -9.6347  -9.6347  -9.3646  -9.3646  -8.4144  -8.4144  -7.2418  -7.2418
    -6.5851  -6.5851  -6.3622  -6.3622  -5.5800  -5.5800  -4.3583  -4.3583
    -0.5040  -0.5040   0.8640   0.8640   1.6397   1.6397   2.0819   2.0819

          k = 0.0833 0.4330 0.0000 ( 14431 PWs)   bands (ev):

   -21.1224 -21.1224 -19.6629 -19.6629 -18.2492 -18.2492 -16.9037 -16.9037
   -14.1172 -14.1172 -12.8567 -12.8567 -11.4637 -11.4637 -11.2740 -11.2740
   -10.1983 -10.1983  -8.9760  -8.9760  -8.2071  -8.2071  -8.0979  -8.0979
    -7.2851  -7.2851  -6.8300  -6.8300  -5.7083  -5.7083  -4.3385  -4.3385
    -0.3553  -0.3553   1.1941   1.1941   2.0876   2.0876   2.8771   2.8772

          k = 0.0833-0.6255 0.0000 ( 14435 PWs)   bands (ev):

   -20.7133 -20.7133 -20.2249 -20.2249 -18.3253 -18.3253 -16.9721 -16.9721
   -13.6179 -13.6179 -12.3757 -12.3757 -11.8264 -11.8264 -10.6670 -10.6670
   -10.3557 -10.3557  -9.2596  -9.2596  -8.4953  -8.4953  -7.9280  -7.9280
    -7.7199  -7.7199  -7.2893  -7.2893  -5.7736  -5.7736  -4.3382  -4.3382
    -0.2565  -0.2565   1.3854   1.3854   2.6468   2.6468   3.3167   3.4385

          k = 0.1667 0.2887 0.0000 ( 14424 PWs)   bands (ev):

   -21.4912 -21.4912 -18.5962 -18.5962 -18.5962 -18.5962 -16.2821 -16.2821
   -15.2702 -15.2702 -13.0401 -13.0401 -13.0400 -13.0400 -10.6236 -10.6236
    -9.4126  -9.4126  -8.9385  -8.9385  -8.9384  -8.9384  -7.0559  -7.0559
    -6.3662  -6.3662  -6.1163  -6.1163  -6.1163  -6.1163  -4.0809  -4.0809
    -0.8286  -0.8286   1.4730   1.4730   1.4731   1.4731   1.5838   1.5839

          k = 0.1667 0.3849 0.0000 ( 14446 PWs)   bands (ev):

   -21.2046 -21.2046 -19.2647 -19.2647 -18.7923 -18.7923 -16.2070 -16.2070
   -14.8968 -14.8968 -12.4402 -12.4402 -11.9750 -11.9750 -11.3833 -11.3833
    -9.3755  -9.3755  -9.0732  -9.0732  -8.6525  -8.6525  -7.8571  -7.8571
    -6.8770  -6.8770  -6.8371  -6.8371  -6.3086  -6.3086  -3.8227  -3.8227
    -0.9383  -0.9383   1.8950   1.8950   1.9759   1.9759   2.5767   2.5767

          k = 0.1667-0.6736 0.0000 ( 14423 PWs)   bands (ev):

   -20.8356 -20.8356 -19.8626 -19.8626 -18.9107 -18.9107 -16.2450 -16.2450
   -14.3710 -14.3710 -12.0733 -12.0733 -11.8864 -11.8864 -10.8930 -10.8930
    -9.8538  -9.8538  -8.7937  -8.7937  -8.6372  -8.6372  -8.2918  -8.2918
    -7.5094  -7.5094  -7.3880  -7.3880  -6.4261  -6.4261  -3.6656  -3.6656
    -0.9783  -0.9783   2.1933   2.1933   2.4797   2.4797   3.2722   3.2723

          k = 0.1667-0.5774 0.0000 ( 14416 PWs)   bands (ev):

   -20.3874 -20.3874 -20.3874 -20.3874 -18.9492 -18.9492 -16.2648 -16.2648
   -14.0752 -14.0752 -12.4189 -12.4189 -11.6086 -11.6086 -10.1087 -10.1087
    -9.8215  -9.8215  -9.8215  -9.8215  -8.1150  -8.1150  -8.1150  -8.1150
    -7.8664  -7.8664  -7.8664  -7.8664  -6.4654  -6.4654  -3.6139  -3.6139
    -0.9851  -0.9851   2.3043   2.3043   3.0945   3.0945   3.0945   3.0945

          k = 0.2500-0.7217 0.0000 ( 14432 PWs)   bands (ev):

   -20.8767 -20.8767 -19.4223 -19.4223 -19.4223 -19.4223 -15.7554 -15.7554
   -14.8774 -14.8774 -12.1516 -12.1516 -11.4022 -11.4022 -11.4021 -11.4021
    -9.0275  -9.0275  -9.0273  -9.0273  -8.6858  -8.6858  -8.5819  -8.5819
    -7.4843  -7.4843  -7.0075  -7.0075  -7.0075  -7.0075  -3.3312  -3.3312
    -1.3369  -1.3369   2.4239   2.4239   2.9761   2.9761   2.9761   2.9761

          k = 0.2500-0.6255 0.0000 ( 14463 PWs)   bands (ev):

   -20.4688 -20.4688 -19.9836 -19.9836 -19.5036 -19.5036 -15.5873 -15.5873
   -14.6316 -14.6316 -12.7161 -12.7161 -10.9648 -10.9648 -10.3881 -10.3881
    -9.4302  -9.4302  -9.3838  -9.3838  -8.5992  -8.5992  -8.2077  -8.2077
    -8.0799  -8.0799  -7.6476  -7.6476  -7.0944  -7.0944  -2.9776  -2.9776
    -1.6237  -1.6237   2.9833   2.9833   3.1990   3.1990   3.6533   3.7782

          k = 0.3333-0.5774 0.0000 ( 14418 PWs)   bands (ev):

   -20.0229 -20.0229 -20.0229 -20.0229 -20.0229 -20.0229 -15.0186 -15.0186
   -15.0166 -15.0166 -12.9829 -12.9829 -10.1411 -10.1411 -10.1411 -10.1411
   -10.1409 -10.1409  -8.7776  -8.7776  -8.7775  -8.7775  -8.7775  -8.7775
    -7.6913  -7.6913  -7.6913  -7.6913  -7.6913  -7.6913  -2.2923  -2.2923
    -2.2922  -2.2922   4.1732   4.1732   4.1732   4.1732   4.1733   4.1733

     the Fermi energy is    -2.2919 ev

!    total energy              =     -96.18460304 Ry
     Harris-Foulkes estimate   =     -96.18460304 Ry
     estimated scf accuracy    <          2.0E-09 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =    -900.03024634 Ry
     hartree contribution      =     455.23975731 Ry
     xc contribution           =     -34.29151362 Ry
     ewald contribution        =     382.89753418 Ry
     smearing contrib. (-TS)   =      -0.00013457 Ry

     convergence has been achieved in   7 iterations

     Writing output data file graphene.save/
 
     init_run     :      4.25s CPU      4.29s WALL (       1 calls)
     electrons    :      6.23s CPU      6.33s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.12s CPU      0.14s WALL (       1 calls)
     potinit      :      0.02s CPU      0.03s WALL (       1 calls)
     hinit0       :      0.04s CPU      0.04s WALL (       1 calls)

     Called by electrons:
     c_bands      :      2.78s CPU      2.82s WALL (       7 calls)
     sum_band     :      0.67s CPU      0.67s WALL (       7 calls)
     v_of_rho     :      2.27s CPU      2.27s WALL (       8 calls)
     mix_rho      :      0.04s CPU      0.04s WALL (       7 calls)

     Called by c_bands:
     init_us_2    :      0.01s CPU      0.01s WALL (      30 calls)
     cegterg      :      1.78s CPU      1.82s WALL (      14 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :      1.25s CPU      1.30s WALL (      67 calls)
     g_psi        :      0.01s CPU      0.01s WALL (      51 calls)
     cdiaghg      :      0.40s CPU      0.40s WALL (      65 calls)

     Called by h_psi:
     h_psi:pot    :      1.25s CPU      1.29s WALL (      67 calls)
     h_psi:calbec :      0.06s CPU      0.07s WALL (      67 calls)
     vloc_psi     :      1.12s CPU      1.15s WALL (      67 calls)
     add_vuspsi   :      0.06s CPU      0.06s WALL (      67 calls)

     General routines
     calbec       :      0.06s CPU      0.07s WALL (      67 calls)
     fft          :      0.03s CPU      0.04s WALL (     101 calls)
     ffts         :      0.01s CPU      0.01s WALL (      28 calls)
     fftw         :      1.21s CPU      1.24s WALL (    7868 calls)
 
     Parallel routines
     fft_scatt_xy :      0.13s CPU      0.13s WALL (    7997 calls)
     fft_scatt_yz :      0.65s CPU      0.67s WALL (    7997 calls)
 
     PWSCF        :     11.54s CPU     16.55s WALL

 
   This run was terminated on:  15: 5:15  15Dec2019            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
